package.json

```diff
@@ -50,7 +54,6 @@
     "./packages/genesys-spark-components",
     "./packages/genesys-spark-chart-components",
     "./packages/*",
-    "./playgrounds/*",
```

Comment:
I'm trying to get the playground workspace to work instead of using the `cd` command to change directory into a playground folder.

Reply:
Our playground project is named `playground` and not `playgrounds` so when I changed the workspace name to be `playground` then running `npm install` on the root of the spark repo was installing angular dependencies into the root's `node_modules` folder which according to Node workspace docs is expected but it was causing some angular reference errors in the root directory. What do you all think? Does it make sense to maybe not have the playgrounds as a workspace?

Reply:
I'm looking into this now, I'll follow up once I find an answer to this

Reply:
I was able to figure this issue out and am using workspaces for the playgrounds now.